zadanie_domowe_MB.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+funkcja_kalkulator () {
+	echo "Podaj pierwsza wartosc liczbowa"
+	read a
+	echo "podaj druga wartosc liczbowa"
+	read b
+	echo "Dodawanie: $((a + b))"
+	echo "Mnożenie: $((a * b))"
+}
+
+funkcja_petla () {
+	echo "Podaj liczbę mniejszą niż 10"
+	read a
+	if (( $a < 10 )); then
+		for (( i = 1; i <= ${a}; i++)); do 
+			echo ${i}
```

Comment:
aby uniknąć błędów przy wyświetlaniu zmiennej zaleca się aby zmienne wypisywać w `"`
```suggestion
			echo "${i}"
```